git_machete/docs.py

```diff
@@ -182,6 +202,36 @@
           <b>--c, --checkout-my-github-prs</b>     Checkout your open PRs into local branches.
           <b>-y, --yes</b>                         Don't ask for confirmation when deleting branches from git.
     """,
+    "config": f"""
+        <b>Config keys: </b>
+        - `machete.github.(remote,organization,repository)`:
+
+          When executing `git machete github <subcommand>` command, {textwrap.indent(github_config_keys, "          ").strip()}
+
+        - `machete.overrideForkPoint.<branch>.(to,whileDescendantOf)`
+
+          Executing `git machete fork-point --override-to=<revision> [<branch>]`, sets up a fork point override for <branch>.
+          The override data is stored under `machete.overrideForkPoint.<branch>.to` and `machete.overrideForkPoint.<branch>.whileDescendantOf` git config keys.
+
+        - `machete.status.extraSpaceBeforeBranchName`
+
+          {textwrap.indent(status_extraSpaceBeforeBranchName_config_key, "          ").strip()}
+
+        - `machete.worktree.useTopLevelMacheteFile`
+
+          # to-do
+
+        <b>Environment variables: </b>
+
+        - `GIT_MACHETE_REBASE_OPTS`
```

Comment:
Also `GIT_MACHETE_EDITOR` is to be documented... pls grep the project for some other env vars but I don't think there are any others